### ISSUE TITLE
Dashboard does not work under Docker

### ISSUE THREAD

**Issue by eriklundberg** [home-assistant/appdaemon#89]
When deploying AppDaemon using Docker, the Dashboard does not load properly and gives a javascript error.

**Steps to reproduce:**
  1. In bash, execute `$ docker run --rm -it -p 5050:5050 -e HA_URL="http://<my_ha_ip>:8123" -e DASH_URL="http://$HOSTNAME:5050" acockburn/appdaemon:latest`
  2. Wait for Docker to download the image and start AppDaemon. The line `INFO Connected to Home Assistant 0.48.1` is written to stdout.
 3. Access `http://localhost:5050/Hello`

The dashboard is all black with the exception of a gray box and the following error is written in Chrome's console: 
```
Uncaught ReferenceError: Unable to process binding "text: function (){return state_text }"
Message: state_text is not defined
    at text (eval at parseBindingsString (knockout-3.4.1.js:68), <anonymous>:3:57)
    at update (knockout-3.4.1.js:98)
    at function.a.B.i (knockout-3.4.1.js:72)
    at Function.Qc (knockout-3.4.1.js:51)
    at Function.Rc (knockout-3.4.1.js:51)
    at Function.aa (knockout-3.4.1.js:50)
    at Object.a.m.a.B (knockout-3.4.1.js:49)
    at knockout-3.4.1.js:72
    at Object.q (knockout-3.4.1.js:11)
    at m (knockout-3.4.1.js:71)
```

**Platform:**
OS: Mac OS Sierra (10.12.5)
Docker: 17.06.0-ce-mac19
Chrome: 59.0.3071.115

**Comment by acockburn** [comment]
Can you post the log output on the screen please. Also, do you have a key set?

**Comment by eriklundberg** [comment]
Just to be sure, do you want the output from the terminal or the javascript console? I'm not running Home Assistant with an API key. 

I tried running the samt configuration outside Docker (natively on my mac) and that works fine.

**Comment by acockburn** [comment]
I'd like to take a look at the terminal output to see how far it is getting.

**Comment by eriklundberg** [comment]
Starting up the container:
```
$ docker run --rm -it -p 5050:5050 -e HA_URL="http://10.0.1.3:8123" -e DASH_URL="http://$HOSTNAME:5050" acockburn/appdaemon:latest
2017-07-19 15:52:53.250307 INFO AppDaemon Version 2.0.4 starting
2017-07-19 15:52:53.250554 INFO Configuration read from: /conf/appdaemon.yaml
2017-07-19 15:52:53.367281 INFO Got initial state
2017-07-19 15:52:53.368392 INFO Loading Module: /conf/apps/hello.py
2017-07-19 15:52:53.372061 INFO Loading Object hello_world using class HelloWorld from module hello
2017-07-19 15:52:53.411141 INFO hello_world: Hello from AppDaemon
2017-07-19 15:52:53.413712 INFO hello_world: You are now ready to run Apps!
2017-07-19 15:52:53.414074 INFO App initialization complete
2017-07-19 15:52:53.415351 INFO Starting dashboard
2017-07-19 15:52:53.429812 INFO Connected to Home Assistant 0.48.1
```

After loading the dashboard the following lines are echoed:
```
2017-07-19 15:53:07.265497 INFO Compiling dashboard 'Hello'
2017-07-19 15:53:07.367851 INFO function [compile_dash] finished in 109 ms
```

**Comment by acockburn** [comment]
OK, thanks - I figured it out and put a fix in 2.0.6 - give it a few minutes and the Docker image should be built.

**Comment by eriklundberg** [comment]
I can confirm that the new image works. Thanks!
